### ISSUE TITLE
Update README with Docker deployment instructions

### ISSUE THREAD

**Issue by prologic** [gilbitron/Raneto#125]
After #124 got landed we should update the docs so other users can easily deploy Raneto.

**Comment by prologic** [comment]
I'll do this even though I can't assign myself issues for a project I have no access to :)

**Comment by ryanlelek** [comment]
Still want to do this @prologic?
Fork repo and add a page to `example/content/`

**Comment by prologic** [comment]
Ahh yes I will :) Hold up!


James Mills / prologic

E: prologic@shortcircuit.net.au
W: prologic.shortcircuit.net.au

**Comment by madhuakula** [comment]
Created an in detailed docker container with completely isolated configuration and markdown files.

We can mount the volumes from local host for Markdown content and Configuration.

https://github.com/appsecco/raneto-docker

**Comment by ryanlelek** [comment]
Added this to the megathread of the Roadmap.

Great link @madhuakula . I'll add it to the Raneto resources on the main README file.

Question:
Where are people deploying Docker apps these days?
- Heroku
- AWS
- DigitalOcean
- Regular/Compose/Podman on a Virtual Machine?

**Comment by ryanlelek** [comment]
Yes, you can update the version in the example repos below to the latest version.
To make a container image run `docker build -t myimage:0.0.1 .` (Search around for tutorials on this)
https://github.com/orgs/raneto/repositories

**Comment by xhj967** [comment]
a lot of thanks, i do it successful ^_^
